### PR TITLE
[2.7][translation] lazy load of standardResources.

### PR DESCRIPTION
**Pull request by aitboudad**
depend on https://github.com/symfony/symfony/pull/14022

### CODE REVIEW

File: Translation/Translator.php [hunk 1]
```diff
@@ -51,12 +52,20 @@ public function addResource($format, $resource, $locale, $domain = null)
...
+     * Return the list of 'standard' resources.
      *
      * @return array
      */
     public function getStandardResources()
     {
+        if (empty($this->standardResources)) {
+            foreach ($this->resourceFiles as $file) {
+                // filename is domain.locale.format
+                list($domain, $locale, $format) = explode('.', basename($file), 3);
```
Comment (dbu) [inline Translation/Translator.php]: travis is not happy with this: https://travis-ci.org/liip/LiipTranslationBundle/jobs/55473320

PHP Warning:  basename() expects parameter 1 to be string, array given in /home/travis/build/liip/LiipTranslationBundle/Translation/Translator.php on line 64

Comment (lsmith77) [inline Translation/Translator.php]: the symfony PR was merged but we still have a regression:
https://travis-ci.org/liip/LiipTranslationBundle/jobs/55473320#L199

**Comment by aitboudad** [comment]
It's because of https://github.com/symfony/symfony/pull/13074

**Comment by lsmith77** [comment]
Build is still failing even though the Symfony PR was merged:
```
PHP Notice:  Undefined property: Liip\TranslationBundle\Translation\Translator::$resourceFiles in /home/travis/build/liip/LiipTranslationBundle/Translation/Translator.php on line 62
```

https://travis-ci.org/liip/LiipTranslationBundle/jobs/55473320#L194

**Comment by aitboudad** [comment]
@lsmith77 fixed but one failure still exist, to fix it we need move [Translator.php#L144-L150](https://github.com/liip/LiipTranslationBundle/blob/master/Translation/Translator.php#L144-L150) into [Translator.php#L126](https://github.com/liip/LiipTranslationBundle/blob/master/Translation/Translator.php#L126) when debug is enabled.

**Comment by lsmith77** [comment]
@aitboudad must admit I do not know this code to well. I guess moving the logic into a separate method then would be the best course of action?

/cc @thormeier

**Comment by aitboudad** [comment]
@lsmith77 me also :), well I'll see if we can fix that in symfony itself

**Comment by aitboudad** [comment]
@lsmith77 the built status now green :)

**Comment by lsmith77** [comment]
and very elegant too!

**Pull request merged**